src/viur/core/modules/file.py

```diff
@@ -449,6 +493,53 @@ def hmac_sign(data: t.Any) -> str:
     def hmac_verify(data: t.Any, signature: str) -> bool:
         return hmac.compare_digest(File.hmac_sign(data.encode("ASCII")), signature)
 
+    @staticmethod
+    def create_serve_parameters(serving_url: str) -> t.Iterable[str] | None:
```

Comment:
Stay conform to your own suggestion below!
```suggestion
    def create_serve_parameters(serving_url: str) -> tuple[str, str]:
```